src/main/java/seedu/planner/model/course/FocusArea.java

```diff
@@ -36,6 +36,10 @@
     private static boolean hasFocusArea(String focusArea) {
         focusArea = focusArea.toLowerCase();
         for (FocusArea fa : FocusArea.values()) {
+            // Do not allow the setting of an {@code UNKNOWN} focus area
+            if (fa.equals(UNKNOWN)) {
+                continue;
```

Comment:
Instead of `continue`, you can just `return false` immediately.
```suggestion
                return false;
```